### ISSUE TITLE
Should Ctrl-z be added to help window?

### ISSUE THREAD

**Issue by functionalCS** [brownplt/code.pyret.org#562]
Ctrl-z (undo), and also Ctrl-x (cut) are browser shortcuts for Edge/Chrome/FF/Opera except Safari where ⌘Z is the text undo shortcut. I have accidentally nuked many programs in the code.pyret.org editor by using Ctrl-A to select all then missing the Tab key and typing a char which of course replaces all selected text with that char. 

Should we add Ctrl-z/⌘Z  or a different undo to here or no https://github.com/brownplt/code.pyret.org/blob/horizon/src/web/editor.html#L378

**Comment by schanzer** [comment]
YES PLEASE! You'd be amazed how many students and teachers don't know that these shortcuts are universal.
